presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -204,6 +211,8 @@ SystemConfig::SystemConfig() {
           STR_PROP(kInternalCommunicationSharedSecret, ""),
           NUM_PROP(kInternalCommunicationJwtExpirationSeconds, 300),
           BOOL_PROP(kUseLegacyArrayAgg, false),
+          STR_PROP(kSinkMaxBufferSize, "32MB"),
+          STR_PROP(kDriverMaxPagePartitioningBufferSize, "32MB"),
```

Comment:
@spershin Do we use default values in cluster setup or we pass them always via config. Asking because if we are using default values, then deployment scripts need to be modified.